docs/source/routing/request-lifecycle.mdx

```diff
@@ -1,22 +1,21 @@
 ---
 title: Router Request Lifecycle
-subtitle: Understand how the router processes client requests
-description: Understand how GraphQL client requests get processed by the request lifecycle pipeline of Apollo GraphOS Router.
+subtitle: Learn how the router processes client requests
+description: Understand how GraphQL client requests get processed through the request lifecycle pipeline of an Apollo Router.
 ---
 
 import RequestLifecycleOverviewDiagram from '../../shared/diagrams/router-request-lifecycle-overview.mdx';
 
-Every client request made to a GraphOS Router goes through the **router request lifecycle**: a multi-stage pipeline of services that processes requests and returns responses.
+Every client request made to an Apollo Router goes through the **router request lifecycle**: a multi-stage pipeline of services that processes requests and returns responses. 
 
 <RequestLifecycleOverviewDiagram />
 
-<Tip>
+The router processes a client request by first passing it between services along the lifecycle's **request path**. In the request path, it needs to figure out how to use your subgraphs to fetch or update the fields of the request. To do this, the router generates a _query plan_:
 
-Understanding the router request lifecycle makes it easier for you to write router customization plugins or configure router telemetry.
+<img src="../images/get-started/qp.svg" class="dark:hidden" />
+<img src="../images/get-started/qp-dark.svg" class="hidden dark:block" />
```

Comment:
Try setting a width so the diagram doesn't look huge on the page, particularly when compared to the mermaid diagram above:

<img width="694" alt="image" src="https://github.com/user-attachments/assets/f766b306-7d7d-4659-9625-e791e681ed1c" />